pkg/client/environments.go

```diff
@@ -57,22 +63,17 @@ func NewPVMClientWithEnv(c *Client, instanceID, instanceName, env string) (*PVMC
 	if err != nil {
 		return nil, err
 	}
-	return NewPVMClient(c, instanceID, instanceName, e["PIEndpoint"])
+	return NewPVMClient(c, instanceID, instanceName, e)
 }
 
-func NewGenericPVMClientWithEnv(c *Client, instanceID, instanceName, env string, piSession *ibmpisession.IBMPISession) (*PVMClient, error) {
-	e, err := GetEnvironment(env)
-	if err != nil {
-		return nil, err
-	}
-	return NewGenericPVMClient(c, instanceID, instanceName, e["PIEndpoint"], piSession)
+func NewGenericPVMClientWithEnv(c *Client, instanceID string, piSession *ibmpisession.IBMPISession) (*PVMClient, error) {
```

Comment:
if it is directly calling `NewGenericPVMClient` then why do we need `NewGenericPVMClientWithEnv`?

Reply:
Agree, the duplicate declaration has been removed. Thanks!